apps/web/frontend/components/addProjectMemberForm.tsx

```diff
@@ -105,20 +105,21 @@ export const AddProjectMemberForm = (props: AddProjectMemberFormProps) => {
     )
 
   return (
-    <form className="flex w-full  items-center gap-2" onSubmit={handleSubmit(handleUserInviteSubmit)}>
-      <Button type="submit" disabled={isSubmitting} variant="primary" className="h-16 w-16 flex-none rounded-full">
-        <AiOutlinePlus className=" h-10 w-10 fill-white" />
-      </Button>
+    <form className="flex items-center gap-2" onSubmit={handleSubmit(handleUserInviteSubmit)}>
       <InputField
+        className="input input-bordered w-full"
```

Comment:
I think the input field should have the input class by default. Maybe input-border as well?